packages/components/src/Table/Table.module.scss

```diff
@@ -80,19 +77,55 @@ $row-height-data-variant: 48px;
 .headerRowCell .headerRowCellTooltip {
   display: flex;
   align-items: stretch;
+  max-width: 100%;
 }
 
-// overflow has to be set at this level as well as on the heading for some reason ¯\_(ツ)_/¯
```

Comment:
the overflow hidden on the tooltip layer was obscuring the focus ring - this is a less intrusive way to let the width of the container be used to set the ellipsis for long labelText